performance/tests/OverallView.test.js

```diff
@@ -0,0 +1,101 @@
+const TestResults = require('../TestResults');
+const {CLS, FID, FPS, getAverageFPS, PageLoadingMetrics} = require('../TraceModel');
+const {clsValue, firstInputValue, getFileName} = require('../utils');
+
+describe('OverallView', () => {
+	const component = 'Overall';
+	TestResults.newFile(component);
+
+	it('FPS', async () => {
+		await FPS();
+		await page.goto('http://localhost:8080/overallView');
+		await page.waitForSelector('#virtualGridList');
+		await page.click('#tooltipButton'); // to move to the next panel.
+		await page.waitForTimeout(200);
+		await page.keyboard.down('Escape');
+		await page.keyboard.up('Escape');
+		await page.waitForTimeout(200);
```

Comment:
Sorry I have one question.
I tried belows but there is error
What did I do wrong?

```suggestion
		await page.click('#tooltipButton'); // to move to the next panel.
		await page.waitForTimeout(200);
		await page.keyboard.down('Escape');
		await page.keyboard.up('Escape');
		await page.waitForTimeout(200);
		
		await page.click('#tooltipButton'); // to move to the next panel.
		await page.waitForTimeout(200);
		await page.keyboard.down('Escape');
		await page.keyboard.up('Escape');
		await page.waitForTimeout(200);

```

```
 FAIL  performance/tests/OverallView.test.js (24.81s)
  OverallView
    ✕ FPS (3165ms)
    ✓ should have a good FID and CLS (2975ms)
    ✓ should have a good DCL, FCP and LCP (17335ms)

  ● OverallView › FPS

    No node found for selector: #tooltipButton

      17 |              await page.waitForTimeout(200);
      18 |
    > 19 |              await page.click('#tooltipButton'); // to move to the next panel.
         |              ^
      20 |              await page.waitForTimeout(200);
      21 |              await page.keyboard.down('Escape');
      22 |              await page.keyboard.up('Escape');

      at Object.assert (node_modules/puppeteer/src/common/assert.ts:23:21)
      at DOMWorld.click (node_modules/puppeteer/src/common/DOMWorld.ts:450:5)
      at Object.<anonymous> (performance/tests/OverallView.test.js:19:3)
```

Reply:
I understand that you want to do a another transition Panel1 -> Panel 2
I believe the timeout of 200ms is not enough. The default duration to complete a transition is 300ms in ViewManager. I suggest using _"await page.waitForSelector('#virtualGridList');"_after using the Escape Key to be sure the test behaves correctly on each environment it is ran.

Reply:
I already tried 

      17 |              await page.waitForTimeout(2000);
      18 |              await page.waitForSelector('#tooltipButton');"
      19 |              await page.click('#tooltipButton'); // to move to the next panel.
      
      But it also have error

    : Timeout - Async callback was not invoked within the 30000ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 30000ms timeout specified by jest.setTimeout.Error:

Reply:
And I got to know below is work.

		await page.click('#tooltipButton');  
		await page.waitForTimeout(2000);  /// When I expand this time, it works
		await page.keyboard.down('Escape');
		await page.keyboard.up('Escape');
		await page.waitForTimeout(200);  

		await page.waitForSelector('#tooltipButton');"
		 await page.click('#tooltipButton'); // to move to the next panel.

As you said, 200 doesn't seem like a safe time.
Instead of delaying a specific ms, please use a more suitable method.

Reply:
await FPS();
		await page.goto('http://localhost:8080/overallView');
		await page.waitForSelector('#virtualGridList');
		await page.click('#tooltipButton'); // to move to the next panel.
		await page.waitForTimeout(200);
		await page.keyboard.down('Escape');
		await page.keyboard.up('Escape');
		await page.waitForSelector('#tooltipButton');
		await page.click('#tooltipButton'); // to move to the next panel.
		await page.waitForTimeout(200);
		await page.keyboard.down('Escape');
		await page.keyboard.up('Escape');
		await page.waitForTimeout(200);

This works. Using waitForSelector instead of waitForTimeout. I'm not sure why these 2 don't get along together :(